documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -890,6 +891,7 @@ a|Mandatory field that describes the source metadata for the event. This field c
 * {prodname} version
 * Connector type and name
 * Database and table that contains the new row
+* Sequence of offset information, including the last committed offset in the database log
```

Comment:
See my comment above re general considerations around describing the format and semantics. I think for each connector we also should state what the individual values represent.